src/bench/bench.h

```diff
@@ -41,6 +41,16 @@ using ankerl::nanobench::Bench;
 
 typedef std::function<void(Bench&)> BenchFunction;
 
+enum BenchPriorityLevel
```

Comment:
I'd use `enum class`

Reply:
think it would be nice if the enum would be some kind of bitmask instead. Then the entries act more like tags, and it would be possible to e.g. run all high and medium benchmarks by specifying both.

Reply:
yeah, much better.
pushed it.